app/routes/circulars._archive._index/route.tsx

```diff
@@ -227,6 +228,7 @@ export default function () {
             name="query"
             type="search"
             defaultValue={inputQuery}
+            value={inputQuery}
```

Comment:
I don't remember where I read this, but I think it's considered a common mistake to have an input that, on change, updates a state variable, _and_ whose value comes from a state variable. @dakota002, @Courey, have you heard that? Or am I misremembering?

Reply:
I'm not sure on that, I think it might be dependent on where in the general hierarchy of components those state values are used. Like if they need to affect a value in a child/parent component. And if they are basic inputs vs more complex components. But I think generally speaking your idea sounds correct

I think there was something going on here where, when the page reloads due to a form submission, the value was not updating correctly, like the `defaultValue` would still show the previous value, even though the query string would update. @tylerbarna is this correct or am I thinking of another thing?

Reply:
@dakota002 without setting `value` the search bar won't update to reflect the current query string inside the search bar until the page is reloaded, so there's not visual feedback that one of the quick action buttons have been pressed until you reload the page.

Reply:
@lpsinger apologies for missing this last week, I'm going to close this PR and open a new one with just the in-page documentation